ui/src/message_popup/author/question_editing_components/learning_objectives.jsx

```diff
@@ -0,0 +1,54 @@
+/* @flow weak */
+import React from 'react';
+
+import {learningObjectives} from '../../../data/learning_objectives.js';
+
+import Paper from 'material-ui/Paper';
+import Divider from 'material-ui/Divider';
+import DropDownMenu from 'material-ui/DropDownMenu';
+import MenuItem from 'material-ui/MenuItem';
+
+export default React.createClass({
+  displayName: 'LearningObjectives',
+
+  propTypes: {
+    learningObjective: React.PropTypes.object.isRequired,
+    onChangeLearningObjective: React.PropTypes.func.isRequired
```

Comment:
Actually the same idea applies to `onChangeIndicator` for the indicator component, so double 😄